src/global.css

```diff
@@ -14,7 +14,7 @@ html {
   box-sizing: inherit;
 }
 
-body {
+/* body {
```

Comment:
There is no need for a `global.css` file if you are using  styled-components. In that case, what you can do is create a global stylesheet via styled-components.

Reference: https://styled-components.com/docs/api#createglobalstyle